src/neo-vm/Unsafe.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("neo-vm.Tests")]

```

Comment:
We usually add this line to `Properties/AssemblyInfo.cs`.

Reply:
There is AssemblyInfo in dotnet core?

Reply:
https://github.com/neo-project/neo/blob/master/neo/Properties/AssemblyInfo.cs